Source/WebKit/NetworkProcess/storage/CacheStorageCache.cpp

```diff
@@ -323,9 +323,9 @@ void CacheStorageCache::putRecordsInStore(Vector<CacheStorageRecord>&& records,
             record.info.insertionTime = existingRecordInfo->insertionTime;
             record.info.url = existingRecordInfo->url;
             record.requestHeadersGuard = existingRecord->requestHeadersGuard;
-            record.request = existingRecord->request;
-            record.options = existingRecord->options;
-            record.referrer = existingRecord->referrer;
+            record.request = WTFMove(existingRecord->request);
+            record.options = WTFMove(existingRecord->options);
+            record.referrer = WTFMove(existingRecord->referrer);
```

Comment:
We fix a UAM but we are doing more move, this seems a bit strange.
In any case, is it always guaranteed that we will not find again the same existingRecords?
It might be clearer if we iterate over existingRecords instead of records, though maybe it is more expensive?

Reply:
Maybe we can improve the naming in a follow-up